docs-content/upgrading.html.md.erb

```diff
@@ -0,0 +1,95 @@
+---
+title: Updating CyberArk Conjur Service Broker for PCF
```

Comment:
URL is `upgrading` but header says `updating`

Reply:
Upgrade or Update (no ing's :) )